addons/website/models/res_config.py

```diff
@@ -16,32 +16,59 @@ def _default_website(self):
     website_name = fields.Char('Website Name', related='website_id.name')
 
     language_ids = fields.Many2many(related='website_id.language_ids', relation='res.lang')
-    default_lang_id = fields.Many2one(related='website_id.default_lang_id', relation='res.lang', required=True)
+    number_of_languages = fields.Integer(string='Number of languages', compute='_compute_number_of_languages', readonly=True)
+    default_lang_id = fields.Many2one(string='Default', related='website_id.default_lang_id', relation='res.lang', required=True)
```

Comment:
string: Default language. If necessary, use a specific label on the settings view if something different is requested. This way the field string is understandable.